README.md

```diff
@@ -1 +1,25 @@
-# hw01
+# Hello There
+
+My name is Allison Leon. I am a fourth year majoring in **Public Policy** with a minor in _Human Rights_. I am originally from *Chicago* and grew up in the **Back of the Yards neighborhood**. I have had experience with R in the past, as it is almost a requirement to familiarize yourself with it for the PubPol major. However, I always felt that those courses brushed through R too quickly, and I wanted to really understand and be able to use R for myself and future work. I am excited to be frustrated by this course, as this first assignment already proved!
+
+Here are a few more facts about me
+* I am Guatemalan
+* I have two younger sisters
+* I find it easiest to study while listening to [instrumental metal music](https://www.youtube.com/watch?v=wPSWsz2R6Xc&t=2857s&ab_channel=Dorkly)
+* I have a dog named Monch (pictured below)
+
+![My dog Monch](https://scontent.xx.fbcdn.net/v/t1.15752-9/62100069_327535848175804_6586088145165484032_n.jpg?_nc_cat=100&_nc_sid=b96e70&_nc_ohc=iCQKhtZDxKEAX8Q3swI&_nc_oc=AQlavXxUo9PsPof-kX7Bg65kiau-RtjOPgr6Q12nCrVW5RBJsZHma21QISSHj6pnf1xaj9VZBknFfDgZSduWlZ7Q&_nc_ad=z-m&_nc_cid=0&_nc_ht=scontent.xx&oh=60202a891e20e6c68f2f0793bc787e91&oe=5F9F3B54)
+
```

Comment:
Great work with adding an image to the introduction. Monch is really cute! About the image size issue, I had the same question as well. One way I tried is to cut down the size of the image before adding it to the file, but that makes the image fuzzier, so maybe not the best way.